layouts/default.vue

```diff
@@ -1,11 +1,117 @@
 <template>
   <div class="container">
+    <Greeting></Greeting>
```

Comment:
The problem is that we need to tell people to remove not only line `3` on that file, but also the preceding comment line 😓